cnames_active.js

```diff
@@ -2284,6 +2284,7 @@ var cnames_active = {
   "sdh": "sdh.netlify.com",
   "sdk-design": "hueitan.github.io/javascript-sdk-design",
   "searchconfig": "tomchen.github.io/searchconfig",
+  "season":"seasonjs.github.io/tools/",
```

Comment:
```suggestion
  "season": "seasonjs.github.io/tools",
```
Space between key & value, remove trailing slash

Reply:
Thank your,and I have fixed it~~